src/net.cpp

```diff
@@ -67,6 +67,15 @@ namespace {
     };
 }
 
+//immutable thread safe array of allowed commands for logging inbound traffic
+const static std::string logAllowIncomingCmds[] = {
+    "version", "verack", "addr", "inv", "getdata",
+    "getblocks", "getheaders", "tx", "headers", "block",
+    "getaddr", "mempool", "ping", "pong", "alert",
+    "filterload", "filteradd", "filterclear", "reject"};
```

Comment:
I noticed some `*other*` in the stats from normal Bitcoin Core client, so I checked all uses of PushMessage:
These need to be added: `, "merkleblock", "notfound"`